### PR TITLE
Added changelog for 3.18.6

### PR DESCRIPTION
**Pull request by larsewi**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,17 @@
+3.18.6:
+	- Cashed policy function results now take into account number of arguments
```
Comment (nickanderson) [inline ChangeLog:2]:
```suggestion
	- Cached policy function results now take into account number of arguments
```

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+	- Modified classesmatching() function to search parent bundles with
+	  inherit => true (ENT-5850)
+	- package[update]smatching now looks for the software inventory databases
+	  in the state directory and uses them if found. This change enables the
+	  usage of these functions in standalone policy files without the demand
+	  for specifying the default package inventory attribute in body common
+	  control. However, you still need the default package inventory attribute
+	  specified in the policy framework for the software inventory databases
+	  to exist in the first place and to be maintained. (ENT-9083)
+
```
Comment (nickanderson) [inline ChangeLog:14]:
```suggestion
	- Fixed misleading error message when adding CMDB classes (ENT-10336)
	- 
```

**Review by nickanderson — CHANGES_REQUESTED** [review]
I see a misspelling and a missing entry for ENT-10336

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,17 @@
+3.18.6:
+	- Cashed policy function results now take into account number of arguments
+	  and function name (CFE-4244)
+	- Improved syntax description for validjson() (ENT-9759)
+	- Modified classesmatching() function to search parent bundles with
+	  inherit => true (ENT-5850)
+	- package[update]smatching now looks for the software inventory databases
```
Comment (olehermanse) [inline ChangeLog:7]:
```suggestion
	- packagesmatching() and packageupdatesmatching() now look for the software inventory databases
```

File: ChangeLog [hunk 4]
```diff
@@ -1,3 +1,17 @@
...
+	  in the state directory and uses them if found. This change enables the
```
Comment (olehermanse) [inline ChangeLog:8]:
```suggestion
	- packagesmatching() and packageupdatesmatching() now look for the software
	  inventory databases in the state directory and use them if found. This change enables the
```

**Review by olehermanse — APPROVED** [review]

**Comment by larsewi** [comment]
Squashed commits

**Review by olehermanse — APPROVED** [review]

**Review by nickanderson — APPROVED** [review]

**Pull request merged**